<launch>
  <node pkg="uvc_camera" type="uvc_camera_node" name="barc_cam">
    <param name="index" value="0"/>
    <param name="device" value="/dev/video0"/>
    <param name="width" value="320"/>
    <param name="height" value="240"/>
  </node>
</launch>
